python/pyspark/sql/functions.py

```diff
@@ -2804,6 +2804,20 @@ def pandas_udf(f=None, returnType=None, functionType=None):
        |  1|1.5|
        |  2|6.0|
        +---+---+
+       >>> @pandas_udf("id long, v1 double, v2 double", PandasUDFType.GROUPED_MAP)  # doctest: +SKIP
```

Comment:
It took me a while to realize `v1` is a grouping key. It also a bit uncommon to use double value as a grouping key . How about we do sth like?

`id long, additional_key long, v double`